src/viam/sdk/components/board.cpp

```diff
@@ -23,7 +23,14 @@ API API::traits<Board>::api() {
 Board::status Board::from_proto(const viam::component::board::v1::Status& proto) {
     Board::status status;
     for (const auto& analog : proto.analogs()) {
-        status.analog_reader_values.emplace(analog.first, analog.second);
+        Board::analog_value result;
+        result.value = analog.second;
+        // The status does not contain the extra data to describe the accuracy of the reader. We
+        // fill those in with 0's instead.
+        result.min_range = 0.0;
+        result.max_range = 0.0;
+        result.step_size = 0.0;
+        status.analog_reader_values.emplace(analog.first, result);
```

Comment:
This can be a one liner:
```
status.analog_reader_values.emplace(analog.first, {analog.second, 0.0, 0.0, 0.0});
```

If you are willing to rely on some C++ initialization nuance, you could I _think_ make this:
```
status.analog_reader_values.emplace(analog.first, {analog.second});
```
which I believe will value initialize the float fields, meaning they become zero.

If that feels too magical, you could also give the non-`value` fields of `analog_value` non-static data member initializers explicitly setting those fields to zero.